tests/TypeAnnotationTest.php

```diff
@@ -595,12 +595,8 @@ class C extends A {}
             ],
             'importedTypeUsedInAssertion' => [
                 'code' => '<?php
-                    /**
-                     * @psalm-type Foo = string
-                     */
-                    class A
-                    {
-                    }
+                    /** @psalm-type Foo = string */
+                    class A {}
```

Comment:
There was nothing wrong with the formatting, but I needed to add some change to re-trigger CircleCI build.